### PR TITLE
Final touch

### PR DESCRIPTION
**Pull request by Apatakiti**
I think I have fulfilled all the necessary procedure
- At this point, you should have a fully working app that uses only basic styles to make the layout work, according to this wireframe.
- Now you should improve the look and feel of the application, adding the styles of your choice.
- Please keep the general layout of the wireframe, this is the only mandatory requirement.
- You can use plain CSS or any CSS framework, it's up to you.

### CODE REVIEW

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @Ademola-coding 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️, but you are almost there! 🏃🏾 

## Highlights
- [x] Linters passed 👍🏾 
- [x] Good PR description 💯 
- [x] Good commit history 👍🏾 

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance to learn practical coding skills and new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them to get your project approved ✅

- [x] Good job for adding a README file, but it's required to fill it with the necessary information to make it look professional. Your README should contain all the needed information for running the app with the API(s) it consumes.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (arslanbisharat) [inline .vscode/settings.json:2]: - [x] [OPTIONAL] Please make sure you don't have unnecessary files in your project director. This makes the project dirty. You can add `.vscode `to `.gitignore`.

**Review by arslanbisharat — APPROVED** [review]
Hi @Ademola-coding,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .vscode/settings.json (hunk 1 above)
Comment (Apatakiti) [reply .vscode/settings.json:2]: thanks

**Pull request merged**